src/Claim/Claims.php

```diff
@@ -58,12 +59,12 @@ private function getGuidKey( $guid ) {
 	}
 
 	/**
-	 * @param Claim $claim
+	 * @param Statement $claim
```

Comment:
What's the purpose of this comment other than making me upset? Do you want me to change the variable names? I will happily do this. Just tell me. But please do not play games.